sql/table.sql

```diff
@@ -1520,7 +1520,7 @@ CREATE TABLE gene_tree_node (
   root_id                         int(10) unsigned,
   left_index                      int(10) NOT NULL DEFAULT 0,
   right_index                     int(10) NOT NULL DEFAULT 0,
-  distance_to_parent              double default 1.0 NOT NULL,
+  distance_to_parent              double default 1 NOT NULL,
```

Comment:
Just a comment to leave here: not sure if the float-point approximation will do its magic when the default is an integer instead of float. But let's hope for the best :+1:

Reply:
About this one, what was the problem here @james-monkeyshines ? Is it that the parser expects at least one non-zero digit after the decimal point ? or that it doesn't support decimal values at all ?

Reply:
It seems as though mysql ignores the decimal bit when creating the table; the datacheck is doing a `show create table` to get the definition, and that has `double default 1 NOT NULL`.